lib/teiserver_web/controllers/clans/clan_controller.ex

```diff
@@ -259,6 +259,11 @@ defmodule TeiserverWeb.Clans.ClanController do
     clan = Clans.get_clan!(clan_id)
 
     cond do
+      user_id == nil ->
+        conn
+        |> put_flash(:danger, "User id missing")
```

Comment:
It's going to be seen by non-developers, I'd rather something more like "Unable to find that user"

Reply:
Changed to "Unable to find user"